### PR TITLE
170 Remodel INCORE home page

### PR DESCRIPTION
**Pull request by Vismayak**
Do you guys have any input on how I can make the page look better? :)

### CODE REVIEW

File: src/components/HomePage.jsx [hunk 1]
```diff
@@ -223,256 +297,253 @@ class HomePage extends Component {
...
+											Sign up
+										</Link>
+										&nbsp;for an IN-CORE account
+									</li>
+									<li>
+										Install&nbsp;
+										<Link href={config.pyIncoreDocUrl} target="_blank" className={classes.link}>
+											pyincore
+										</Link>
+										&nbsp;pyincore, a python package that contains service classes to connect with
```
Comment (longshuicy) [inline src/components/HomePage.jsx:431]: "pyincore" is repeated here
image

File: src/components/HomePage.jsx [hunk 2]
```diff
@@ -223,256 +297,253 @@ class HomePage extends Component {
...
+									</Link>
+									.
+								</p>
+								<div className={classes.buttonDiv}>
+									<Button
+										variant="contained"
+										color="primary"
+										alignSelf="center"
+										href={config.incoreDocUrl}
+										className={classes.button}
```
Comment (longshuicy) [inline src/components/HomePage.jsx:474]: new tab

File: src/components/HomePage.jsx [hunk 3]
```diff
@@ -223,256 +297,253 @@ class HomePage extends Component {
...
+									>
+										IN-CORE Manual
+									</Button>
+									<Button
+										variant="contained"
+										color="primary"
+										alignSelf="center"
+										href={config.incoreTutorialUrl}
+										className={classes.button}
+										startIcon={<SchoolIcon />}
```
Comment (longshuicy) [inline src/components/HomePage.jsx:485]: open in new tab

File: src/components/HomePage.jsx [hunk 4]
```diff
@@ -223,256 +297,253 @@ class HomePage extends Component {
...
+										variant="contained"
+										color="primary"
 										className={classes.button}
-										style={{marginRight: "1em"}}
-								>
-									<img src="/public/slack-logo.png" style={{ height: "1em", marginRight: "8px"}}/>
-									Join Slack
-								</Button>
-								<Button href={`mailto:${config.mailingList}`}
+										style={{ marginRight: "1em" }}
```
Comment (longshuicy) [inline src/components/HomePage.jsx:525]: You might want to remove the marginRight to be consistent with the space in between 2 buttons
image

File: src/components/HomePage.jsx [hunk 5]
```diff
@@ -223,256 +297,253 @@ class HomePage extends Component {
...
+									</Button>
+								</div>
+								<Divider />
+							</div>
+							<div>
+								<h2>Get Help</h2>
+								<p>
+									Got questions or need assistance? There are three ways to reach out for help:&nbsp;
+									<Link href={config.incoreFAQUrl} target="_blank" className={classes.link}>
+										review our FAQs for the most common questions
```
Comment (longshuicy) [inline src/components/HomePage.jsx:497]: I would put the hyperlink only around the "FAQs"

File: src/components/HomePage.jsx [hunk 6]
```diff
@@ -223,256 +297,253 @@ class HomePage extends Component {
...
+									</Link>
+									, &nbsp;
+									<Link href={config.slackInvitationLink} target="_blank" className={classes.link}>
+										join the IN-CORE Slack channel
```
Comment (longshuicy) [inline src/components/HomePage.jsx:501]: Same only around the "IN-CORE slack channel"

File: src/components/HomePage.jsx [hunk 7]
```diff
@@ -223,256 +297,253 @@ class HomePage extends Component {
...
+									</Link>
+									, or&nbsp;
+									<Link href={`mailto:${config.mailingList}`} className={classes.link}>
```
Comment (longshuicy) [inline src/components/HomePage.jsx:504]: Only around "email"?

File: src/components/HomePage.jsx [hunk 8]
```diff
@@ -223,256 +297,253 @@ class HomePage extends Component {
...
-												{option}
-											</Link>
-										))}
-									</Typography>
+									>
+										Email Us
+									</Button>
 								</div>
-							))}
+								<Divider />
```
Comment (longshuicy) [inline src/components/HomePage.jsx:543]: I would remove the last divider since it's the bottom of the screen. But up to you.

**Review by longshuicy — CHANGES_REQUESTED** [review]
- [ ] - Could you add a changelog entry?
- [ ] - Once expand I still see the logo overlapping the topbar. Could you check if you pushed your fix to that?
- [ ] image
- [ ] - Could you unify the font? Either pick the same as the description; or the same as the instruction.
- [ ] - Could you double check all the buttons/links leads to a new tab so user can still stay on the same page
- [ ]  Since there are less content on the page now maybe we can increase the fontsize? thoughts?

**Review by Rashmil-1999 — APPROVED** [review]
Looks good!

**Review by longshuicy — APPROVED** [review]

**Pull request merged**